source/documentation/modules/math.html.md.erb

```diff
@@ -0,0 +1,494 @@
+---
+title: sass:math
+---
+
+<%= partial '../snippets/built-in-module-status' %>
+
+## Variables
+
+<% function 'math.$e' do %>
+  Equal to the value of the mathematical constant `e`.
```

Comment:
All the new members here should have their own `impl_status` tags indicating when they were added to Dart Sass.

Reply:
Done.